06-best-practices/code/requirements.txt

```diff
@@ -0,0 +1,10 @@
+pipenv
```

Comment:
We should try to find a way to do it without requirements because we already have pipenv

What was the problem you had when using pipenv?